app/view_models/insights_view_model.rb

```diff
@@ -7,8 +7,84 @@ def upcoming_count
     AuctionQuery.new.upcoming_auction_count
   end
 
-  def total_auctions
-    published_auction_count
+  def hero_metrics
+    [
+      published_auction_stat,
+      accepted_auction_stat,
+      unique_winners_stat,
+      average_bids_stat,
+      unique_bidders_stat,
+      vendors_with_bids_stat,
+      average_auction_length_stat,
+      { statistic: average_delivery_time, label: 'average delivery time' },
+      { statistic: average_starting_price, label: 'average starting price' },
+      average_winning_bid_stat,
+      { statistic: small_business_count, label: 'small businesses registered' },
+      { statistic: in_sam_count, label: 'Sam.gov qualified vendors' }
+    ]
+  end
+
+  def published_auctions_stat
+    {
+      statistic: published_auction_count,
+      label: 'total auctions',
+      href: 'chart-bids-by-auction'
+    }
+  end
+
+  def accepted_auction_stat
+    {
+      statistic: accepted_auctions_count,
+      label: 'successful delieries',
+      label_stat: "(#{accepted_auction_percent} success rate)"
+    }
+  end
+
+  def unique_winners_stat
+    {
+      statistic: unique_auction_winners,
+      label: 'unique auction winners',
+      href: 'chart-winning-bid'
+    }
+  end
+
+  def average_bids_stat
+    {
+      statistic: average_bids_per_auction,
+      label: 'bids/auction',
+      href: 'chart4'
+    }
+  end
+
+  def unique_bidders_stat
+    {
+      statistic: unique_bidders_per_auction,
+      label: 'unique bidders per auction'
+    }
+  end
+
+  def vendors_with_bids_stat
+    {
+      statistic: vendors_with_bids_count,
+      href: 'agency-savings',
+      label: 'vendors who have placed bids'
+    }
+  end
+
+  def average_auction_length_stat
+    {
+      statistic: average_auction_length,
+      href: 'auction-length',
+      label: 'average auction length'
+    }
+  end
+
+  def average_winning_bid_stat
+    {
+      href: 'chart-winning-bid',
+      statistic: average_winning_bid,
```

Comment:
Yes, the more i think about it the more complicated/rich it becomes. We can
talk about this in grooming

On Tue, Jul 5, 2016 at 6:21 PM, Jacob Harris <notifications@github.com>
wrote:

> In app/view_models/insights_view_model.rb
> <https://github.com/18F/micropurchase/pull/824#discussion_r69663985>:
>
> > +      label: 'vendors who have placed bids'
> > +    }
> > +  end
> > +
> > +  def average_auction_length_stat
> > +    {
> > +      statistic: average_auction_length,
> > +      href: 'auction-length',
> > +      label: 'average auction length'
> > +    }
> > +  end
> > +
> > +  def average_winning_bid_stat
> > +    {
> > +      href: 'chart-winning-bid',
> > +      statistic: average_winning_bid,
>
> Sometimes I wonder if we should calculate the average decline of the
> winning bid from the starting price as a useful metric (since this will be
> skewed once we start with > 3500 starting amounts). Or maybe median winning
> bid or maybe percent decline. I dunno.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/micropurchase/pull/824/files/a6aafe5ad12de9d1ecf3c10ec3a30ae5d3f831de#r69663985>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGHNWciwtkkdW10K_s04g-9zCEfd15Lks5qSwMigaJpZM4JFodZ>
> .
>



-- 
Michael Torres
Product Lead, Micropurchase Platform
18F
(415) 920-3205